components/Home/AboutUsSection.tsx

```diff
@@ -1,52 +1,81 @@
 import { FC } from 'react';
 
+import Container from '@mui/material/Container';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+
 import Image from 'next/image';
 import Deerfield from 'public/deerfield.png'; // image from https://www.soundsolutions.ca/projects/featured-exterior-cladding-projects/deerfield-hall-univeristy-of-toronto-mississauga-campus
 import Experience from 'public/experience.png';
 
+const theme = createTheme({
+  typography: {
+    fontFamily: ['Segoe UI'].join(','),
+  },
+});
+
 const AboutUsSection: FC = () => (
-  <div
-    id="about"
-    className="about-us-section flex flex-col items-center lg:items-stretch lg:flex-row gap-8"
-  >
-    <div className="header-img w-full lg:w-1/3 lg:min-h-full hidden lg:block">
-      <Image
-        src={Deerfield}
-        alt="Deerfield Hall"
-        className="min-h-full rounded-lg overflow-hidden"
-        draggable="false"
-      />
-    </div>
-    <div className="w-4/5 mt-10 lg:mt-0 lg:w-2/3 flex flex-col justify-between xl:justify-center">
-      <span>
-        <p className="inline-block h-1/3 mr-5 text-2xl">ABOUT US</p>
-        <hr className="inline-block h-1 w-56 bg-purple-600" />
-      </span>
-      <div className="flex items-center my-5 lg:my-0">
-        <div className="w-1/2">
-          <p className="text-2xl md:text-4xl mt-3 mb-3 lg:text-6xl font-bold">FOR THE STUDENTS</p>
-          <p className="lg:hidden text-lg text-purple-700 font-semibold">Academic Society</p>
+  <ThemeProvider theme={theme}>
+    <Container
```

Comment:
as for the theme, we already have a theme defined in the [themes file](https://github.com/utmmcss/mcss-website-frontend/blob/master/style/theme.ts) so we can unify the theming throughout the app